dlc/src/main/scala/org/bitcoins/dlc/DLCOutcome.scala

```diff
@@ -7,7 +7,7 @@ import org.bitcoins.core.wallet.utxo.BitcoinUTXOSpendingInfoFull
 case class DLCOutcome(
     fundingTx: Transaction,
     cet: Transaction,
-    closingTx: Transaction,
+    closingTxOpt: Option[Transaction],
```

Comment:
So what exactly does this state mean? Does this mean we cannot ultimately settle the DLC because the payout is too small?